README.md

```diff
@@ -105,72 +116,109 @@ static String eWONCertificatePath = "/usr/Certificates";
    },
    "eWONConfig":{
-      "CertificatePath":"/usr/Certificates",
-      "FtpUser":"ftpUsernameExample",
-      "FtpPassword":"ftpPasswordExample"
+      "CertificatePath":"/usr/Certificates"
    },
    "AppConfig":{
-      "CommunicationType":"omf"
+      "CommunicationType":"omf",
+      "OSIsoftTagNamingScheme":"default",
+      "LoggingLevel": 4,
+      "httpTimeoutSeconds": 2
    }
 }
 ```
 ### ServerConfig
 
 The OSIsoft PI Database Web API must be installed for this connector to work.  The connector requires three items to connect to the OSIsoft PI Web API they are the URL of the server running the PI Web API, WebID of the PI Web API and a valid user name and password for the system running the PI Web API software.
 
-URL – IP address or domain name of the system running the Web API (in this example assume the IP address is 192.168.0.124)
+#### URL
+IP address or domain name of the system running the Web API (in this example assume the IP address is 192.168.0.124)
 
-WebID – WebID of your OSIsoft Web API.  This value can be attained by pointing a browser to the OSIsoft PIWEBAPI URL (https://192.168.0.124/piwebapi/dataservers).  The WebID is one of several parameters returned.  A username and password will be required to access the page.  If a username is not required it may be cached.  Try clearing the cache to verify the username and password.  The same username and password will be used to create the credentials in the next step.  (A security warning may occur when trying to access this page, this warning should be ignored)
+#### WebID
+This field is only needed when the CommunicationType is set to "piwebapi". WebID is the web ID of your OSIsoft PI Web API.
+This value can be attained by pointing a browser to the OSIsoft PIWEBAPI URL (https://192.168.0.124/piwebapi/dataservers).  The WebID is one of several parameters returned.  A username and password will be required to access the page.  If a username is not required it may be cached.  Try clearing the cache to verify the username and password.  The same username and password will be used to create the credentials in the next step. 
+(If the PIWEBAPI certificates are self-signed a security warning may occur when trying to access this page. This warning should be ignored)
 
 #### Proxy URL
 
-Adding the field "ProxyURL" under the server config section will change request endpoints to https://domain-name/proxy-path. This field is optional. 
-Below is an example config file with the optional proxy URL in use.
-   ```
+This optional field is only needed when using a non default endpoint of https://HOSTNAME/piwebapi. Adding the field "ProxyURL" under the server config section will change request endpoints to https://HOSTNAME/proxy-path. HOSTNAME will be retrieved from the URL field in the configuration file. 
+
+Below is a partial config file with the optional proxy URL in use. Its endpoint is set to https://192.168.1.1/long/proxy/example/omf
+   ```json
    {
    "ServerConfig":{
```

Comment:
Not part of this PR but this should be indented 1 level.

Reply:
@oliver-walker-hms I would add a commit to fix indentation on JSON in README.md...looking at the JSON right now makes it a bit harder to follow with improper indentation on a few portions.